modules/cloudsql-instance/variables.tf

```diff
@@ -204,6 +204,19 @@ variable "network_config" {
   }
 }
 
+variable "password_validation_policy" {
+  description = "Password validation policy configuration for instances."
+  type = object({
+    enabled = optional(bool, true)
```

Comment:
Do you need `enabled`?. I prefer inferring it with `var.password_validation_policy != null`?

Reply:
it's explicit in the API and I think it's done to preserve your policy without enforcing it, I'd live it there